src/main/java/com/scriptbasic/syntax/BasicSyntaxAnalyzer.java

```diff
@@ -1,6 +1,12 @@
 package com.scriptbasic.syntax;
 
-import com.scriptbasic.interfaces.*;
+import com.scriptbasic.interfaces.AnalysisException;
```

Comment:
I do not like wild card imports, but ScriptBasic uses that. I believe that consistency in code is very important. If you want to change all import wildcards to import statements that individually import the classes I am happy with that. The only caveat is that it should be a separate pull request and should do it for all files.

Btw: it is fairly easy to change if you have IntelliJ. IntelliJ does that for you in a single menu click.

As for this pull request, as usual for all pull requests: keep it to contain only the changes that are strictly related to the actual request.